### ISSUE TITLE
give the scaled size to customize the component more simply

### ISSUE THREAD

**Issue by guillaumecrespel** [unirakun/react-kgraph#36]
Is node exists with x/y scaled? no
So the link does not have to.

edit: at least in this pr

_Originally posted by @fabienjuif in https://github.com/unirakun/react-kgraph/pull/30_

**Comment by fabienjuif** [comment]
the size is already given, be more specific in this issue please

**Comment by fabienjuif** [comment]
The x and y positions are the position the layout engine uses.
Theses positions are not scaled because we found out that webcola works better on a tiny viewport.

We then use `size` to scale out the viewport and play with zoom and/or offsets is a easier way.

If the same variable (eg: source.x) has two meanings : one for the layout, one for the graphical engine, it will be hard to follow when we will debug behaviours. **Is this source.x scaled or not?**

This variable (eg: source.x) would be a scaled x for the Link (PR #30) but not in input of `Graph.links` : this is hard to follow and understand.

Right now the solution is to keep it at is since this is not hard to scale the positions and we have some examples in storybook.

**A better solution would be removing all this scaling thing.**
